src/applications/ivc-champva/10-7959C/components/FileUploadWrapper.jsx

```diff
@@ -1,8 +1,14 @@
 import FileFieldCustom from '../../shared/components/fileUploads/FileUpload';
-import { requiredFiles } from '../config/constants';
+import { requiredFiles, UPLOADS_COMPLETE_PATH } from '../config/constants';
```

Comment:
This `UPLOADS_COMPLETE_PATH` constant can be provided on a per-form basis to any forms that use the `MissingFileOverview` (currently just 10-10d and 10-7959c)